Image-Capture-POC/Image-Capture-POC/Protocols/Router.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Router.swift
+//  Image-Capture-POC
+//
+//  Created by Jose Luna on 9/26/25.
+//
+
+import UIKit
+
+protocol Router {
+    associatedtype Route
+    
+    var navigationController: UINavigationController? { get }
+    
+    func process(_ route: Route)
```

Comment:
Necesitas ser mas descriptivo, en este caso no veo la razon de omitir el parametro

```suggestion
    func process(route: Route)
```